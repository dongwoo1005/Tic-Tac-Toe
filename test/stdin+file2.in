game movesX2.txt stdin
S
SW
SE
C
NE
W
NW
N
E
quit
