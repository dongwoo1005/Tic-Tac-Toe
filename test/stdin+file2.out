X's move
(plays C)
O's move
X's move
(plays NE)
O's move
X's move
(plays W)
O's move
O wins
Score is
X 0
O 1
